content/kapp-controller/docs/latest/air-gapped-workflow.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Air-gapped Workflow
+---
+
+The documentation below covers topics from the [imgpkg air-gapped workflow docs](/imgpkg/docs/latest/air-gapped-workflow) 
+more concisely in order to focus on applying these workflows to kapp-controller package repositories. 
+
+## Scenario
+
+You have a [PackageRepository](packaging#packagerepository-cr) in an [imgpkg bundle format](/imgpkg/docs/latest/resources/#bundle) 
+in an external OCI registry that you would like to move into an OCI registry in an air-gapped environment. Once relocated, you would 
+like to deploy the bundle as part of a PackageRepository to a Kubernetes cluster.
+
+## Prerequisites
+
+In order to go through this process of moving an imgpkg bundle to an air-gapped environment, you will need to have [imgpkg](/imgpkg) 
+installed. More information on installing Carvel tools, including `imgpkg`, can be found [here](/#whole-suite).
+
+## Copy PackageRepository bundle to new location
+
+Most of the steps documented for the [imgpkg air-gapped workflow docs](/imgpkg/docs/latest/air-gapped-workflow#step-1-finding-bundle-in-source-registry) 
+still apply in the case of working with kapp-controller package repositories. A summary of these docs is that you will need to copy your package repository 
+bundle with `imgpkg` via one of the following options:
+
+- Option 1: From a common location connected to both registries. This option is more efficient because only changed image layers will be transfered between registries.
+- Option 2: With intermediate tarball. This option works best when registries have no common network access.
+
+More detailed documents for [`Option 1`](/imgpkg/docs/latest/air-gapped-workflow/#option-1-from-a-location-connected-to-both-registries) and 
+[`Option 2`](/imgpkg/docs/latest/air-gapped-workflow/#option-2-with-intermediate-tarball) can be found at the attached links. 
+
+A summary of steps for relocating a package repository bundle to an air-gapped environment are documented for both options below:
+
+For `Option 1`: 
+* Get to a location that can access both registries. If there is no such location, you will have to use `Option 2` steps.
+* [Authenticate](/imgpkg/docs/latest/auth.md) with both source and destination registries
+* Run `imgpkg copy -b index.docker.io/user1/simple-app-bundle:v1.0.0 --to-repo final-registry.corp.com/apps/simple-app-bundle`
+
+For `Option 2`:
+* Get to a location that can access the source registry
+* [Authenticate] with the source registry(/imgpkg/docs/latest/auth.md)
+* Run `imgpkg copy -b index.docker.io/user1/simple-app-bundle:v1.0.0 --to-tar /tmp/my-image.tar`
+* Run `imgpkg copy --tar /tmp/my-image.tar --to-repo final-registry.corp.com/apps/simple-app-bundle`
```

Comment:
do we want to put a step in between that calls out connecting to and authenticating with the destination registry?

Reply:
Yup, makes sense. Added.